source/optouts/models/ErrorResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System.Xml.Serialization;
+
+namespace com.esendex.sdk.optouts.models
+{
+    public class ErrorResponse

```

Comment:
is it worth giving this a more specific name or moving it to a more common place?